### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by belikegt**
got the basics of forking and pull requests!

### CODE REVIEW

File: giftsonhelloworld.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (Rangeeshar) [inline giftsonhelloworld.py]: So whenever you right a python file follow the below things.
The python file should start like this alwats
"""
whatever your script does definition goes here.
"""
if its a script use a function and call a function always.

Comment (Rangeeshar) [reply giftsonhelloworld.py]: do all the print in single line.

**Review by Rangeeshar — CHANGES_REQUESTED** [review]
changes needed

File: giftsonhelloworld.py (hunk 1 above)
Comment (Rangeeshar) [reply giftsonhelloworld.py]: @belikegt  please create a folder of your name and in it add this file

File: Giftson/helloworld.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+'''Python Program to
```
Comment (Rangeeshar) [inline Giftson/helloworld.py]: ''' -> string
""" -> Doc string, always add doc string

**Review by Rangeeshar — CHANGES_REQUESTED** [review]
changes required

**Review by belikegt — COMMENTED** [review]
Comment for the program provided
printed all the statements in a single line

File: Giftson/helloworld.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+"""Python Program to
```
Comment (Rangeeshar) [inline Giftson/helloworld.py:1]: https://www.geeksforgeeks.org/python-code-formatting-using-black/

Refer this and run it your repo and check

**Review by Rangeeshar — CHANGES_REQUESTED** [review]

**Review by Rangeeshar — APPROVED** [review]

**Comment by belikegt** [comment]
Done

**Review by belikegt — COMMENTED** [review]
done? iguess?

**Pull request merged**